src/auth/maptiler_hmacsha256/gui/qgsauthmaptilerhmacsha256edit.cpp

```diff
@@ -14,21 +14,24 @@
  *                                                                         *
  ***************************************************************************/
 
-#include "qgsauthhmacsha256edit.h"
-#include "ui_qgsauthhmacsha256edit.h"
+#include "qgsauthmaptilerhmacsha256edit.h"
+#include "ui_qgsauthmaptilerhmacsha256edit.h"
 
 
-QgsAuthHmacSha256Edit::QgsAuthHmacSha256Edit( QWidget *parent )
+QgsAuthMapTilerHmacSha256Edit::QgsAuthMapTilerHmacSha256Edit( QWidget *parent )
   : QgsAuthMethodEdit( parent )
 {
   setupUi( this );
-  connect( mKeyEdit, &QLineEdit::textChanged, this, &QgsAuthHmacSha256Edit::configChanged );
-  connect( mTokenEdit, &QPlainTextEdit::textChanged, this, &QgsAuthHmacSha256Edit::configChanged );
+  mLinkLabel->setText( "Secure Token from <a href=\"https://cloud.maptiler.com/account/credentials/\">MapTiler Account Credentials</a>" );
+  mLinkLabel->setTextFormat( Qt::RichText );
+  mLinkLabel->setTextInteractionFlags( Qt::TextBrowserInteraction );
+  mLinkLabel->setOpenExternalLinks( true );
```

Comment:
Isn't it possible to set up these properties in the .ui file?

Reply:
ok